tests/test_json_model.py

```diff
@@ -950,9 +950,21 @@ class TypeWithUuid(JsonModel):
 
     await item.save()
 
+@py_test_mark_asyncio
+async def test_return_specified_fields(members, m):
+    member1, member2, member3 = members
+    actual = await m.Member.find(
```

Comment:
One issue to contend with here is that the model validation will not work correctly when running find, you'll need to make sure you return a dictionary or some sub-set of the model to prevent validation errors from being tossed by the find.